### ISSUE TITLE
Not able to distinguish bewteen runner names in report

### ISSUE THREAD

**Issue by mj-moor** [trivago/cluecumber#164]
**Describe the bug**
Hi laxersaz,

I have multiple runners that are executed in the cloud. I.e. 4 iPads en 3 iPhones. All these runners execute the same tests in parallel. After my tests have been completed I get the Cluecumber report showing the results of all my runners. However I don't seem to be able to distinguish based on the runner name name in my report. In other words I do not know which particular scenario has ran by which particular runner. Is there a way to add the runner information or some other variable to the scenarios so that I can filter based on this variable?

**Comment by laxersaz** [comment]
Are your runners creating separate json files? Because in the next release, Cluecumber is supposed to have a new view to group scenarios by file (#149). Until then, maybe it would make sense to either
- generate separate reports for each device
- add the device name to the scenario name
- add the device name as a scenario text attachment
- add a tag for each device

**Comment by mjm987mj** [comment]
Thanks for the response. Yes my runners are creating seperate json files. Nice to hear that within the next release we can group them by file. That would be really helpful. Also thanks for the additional tips.

Do you have an ETA on the next release?

Thanks in advance

**Comment by laxersaz** [comment]
I cannot give you an ETA yet. I started working on a big next release that changes the architecture of Cluecumber so features like this can be implemented much easier and quicker.

**Comment by mjm987mj** [comment]
Hi @laxersaz do you perhaps have an update on the above?

Thanks in advance,

**Comment by laxersaz** [comment]
I had to pause further development for a while but I am planning on picking up some Cluecumber tasks again soon.

**Comment by bischoffdev** [comment]
Duplicate of #149
